src/skills.js

```diff
@@ -1,12 +1,13 @@
 function processData(response) {
-    const skills = response.skills;
-    let output = "";
-    document.getElementById("title3").innerHTML = "SKILLS";
-    for (let i = 0; i < skills.length; i++) {
-      output += skills[i] + "<p>";
-    }
-    document.getElementById("list3").innerHTML = output;
+  const skills = response.skills;
+  document.getElementById("skills-title").innerHTML = "SKILLS";
+  for (let i = 0; i < skills.length; i++) {
+    const myList = document.createElement("li");
+    let listArray = document.getElementById("skills-list");
+    listArray
+      .appendChild(myList)
+      .insertAdjacentHTML("afterbegin", `${skills[i]}`);
```

Comment:
The backticks are something called string interpolation. It basically allows you to mix strings with variables easily.  But it is also costly, from a code execution perspective.  If you aren't added extra strings in between variables, don't use backticks. `.insertAdjacentHTML("afterbegin", skills[i]);` is a more performant way to achieve this.

Likewise, you have this above `${bio[0]}${bio[1]}${bio[2]}` whereas `bio[0] + bio[1] + bio[2]` is the same thing, but takes less processing power to accomplish.